modules/post/windows/recon/wql_query.rb

```diff
@@ -0,0 +1,76 @@
+# -*- coding: binary -*-
+
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  include Msf::Post::Windows::WMIC
+
+  def initialize(info={})
+    super( update_info( info,
+        'Name'          => 'WQL Command Runner',
+        'Description'   => %q{
+          This module will execute a WQL query via Powershell's Get-WmiObject. A WQL query
+          can be explicity defined with the WQL and NAMESPACE options, or a hard-coded
+          query can be used by defining the an ACTION option. The following ACTIONs are
+          available: OSVERSION - Returns information about the operating system,
+          PROCESSINFO - Returns running processes, RAW_QUERY - Run a raw WQL query.
+        },
+        'License'       => MSF_LICENSE,
+        'Author'        => [ 'Chris Higgins' ], # msf module - @ch1gg1ns
+        'Platform'      => [ 'win' ],
+        'SessionTypes'  => [ 'meterpreter' ],
+        'Actions'       =>
+        [
+          ['OSVERSION'],
```

Comment:
Admittedly, these are simple, but the modules I saw that used this added a description.  Not particularly passionate about it in this case, though.